.storybook/components/ColorSwatches/ColorSwatches.tsx

```diff
@@ -1,48 +1,52 @@
 import React from "react";
 import { Button } from "@jobber/components/Button";
+import { Tooltip } from "@jobber/components/Tooltip";
+import { Flex } from "@jobber/components/Flex";
+import { Text } from "@jobber/components/Text";
+import { Content } from "@jobber/components/Content";
 import { showToast } from "@jobber/components/Toast";
 import styles from "./ColorSwatches.module.css";
 
 interface ColorSwatchesProps {
   readonly colors: string[];
+  readonly description?: string;
 }
 
 export function ColorSwatches({ colors }: ColorSwatchesProps) {
   return (
-    <div className={styles.colorbar}>
+    <Content>
       {colors.map((color: string) => (
         <Color key={color} color={color} />
       ))}
-    </div>
+    </Content>
   );
 }
 
 interface ColorProps {
   readonly color: string;
+  readonly description?: string;
 }
 
-function Color({ color }: ColorProps) {
-  const colorsWithBorders = [
-    "--color-overlay--dimmed",
-    "--color-surface",
-    "--color-text--reverse",
-  ];
+function Color({ color, description }: ColorProps) {
+
   const style = {
     backgroundColor: `var(${color})`,
-    border: colorsWithBorders.includes(color)
-      ? "1px solid var(--color-border)"
-      : undefined,
   };
 
   return (
-    <div className={styles.color}>
+    <Flex gap="small" align="center" template={["shrink", "shrink"]}>
       <div key={color} style={style} className={styles.swatch}>
-        <div className={styles.button}>
-          <Button size="small" icon="copy" onClick={handleClick} ariaLabel="Copy" />
-        </div>
       </div>
-      <pre className={styles.pre}>{color}</pre>
-    </div>
+      <Content spacing="small">
+        { description && (<Text>{description}</Text>) }
```

Comment:
This slot and prop will allow us to pass in the description of each token from the json file once added